stdlib/include/llvm/Support/type_traits.h

```diff
@@ -1,4 +1,4 @@
-//===- llvm/Support/type_traits.h - Simplfied type traits -------*- C++ -*-===//
+//===- llvm/Support/type_traits.h - Simplified type traits -------*- C++ -*-===//
```

Comment:
This may violate a column limit. My guess is that I'd steal a `-` from after `traits`.